src/main/java/com/genesys/workspace/models/targets/availability/TargetAvailability.java

```diff
@@ -0,0 +1,5 @@
+package com.genesys.workspace.models.targets.availability;
+
+public class TargetAvailability {
```

Comment:
Looks good - could be abstract?

Reply:
ok then targets without availability would have null as availabily value